onos-classic/values.yaml

```diff
@@ -31,6 +31,11 @@ resources:
 
 individualOpenFlowNodePorts: false
 
+# Adds an additional nodeport to allow profiler connection.
+# When false, profiler can still connect through proxy/port-forward
+profiler:
+  enabled: false
```

Comment:
The name is confusing. Profiler is always enabled when using profiler image. What we control here is whether to further expose a nodePort or not. I'd suggest renaming this to `nodePort: true`

Reply:
Or even more specific: `exposeNodePort: true`